docs/extras/permissions/permissions.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Permissions
+sidebar_position: 1
+---
+
+# Permissions
+
+At DevCycle, we offer Permissions gating to protect Environments which are marked as Production from unauthorized changes. This allows Organizations to manage which users can modify the Targeting Status, Rules and Served Variations in a Production Environment that may impact their End Users.
+
+Permissions can be enabled for Organizations on our Business or Enterprise plans. Please contact our [support](mailto:support@devcycle.com) team to get started.
+
+:::info
+Permissions enforcement will apply to all Projects and Production-type Environments within your Organization.
+:::
+
+## What Can Users Do?
+
+For more information, checkout the [Organization Roles](/essentials/organizations#organization-roles) documentation.
+
+### Members
+
+- Configure Development and Staging Environments
+- Configure **Inactive** Production Environments
+
+### Publishers
+
+Everything **Members** can do, plus
+
+- Change Targeting Status of Production Environments
```

Comment:
will people really know what this means? Is it simpler to just say "turn features on and off" or something

Reply:
good point, how does this sound? `Enable or Disable Production Environment Targeting`

Reply:
better, though I also worry that "disable production targeting" could be interpreted to mean "there are no rules now its just on or off for everyone"

Reply:
I think that's an unlikely interpretation, so i'm not too worried about it